1566771941590	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1566772452660	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1566772452660	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1566772452661	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1566772453660	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1566773021762	Sync.Service	DEBUG	User-Agent: Firefox/68.0.1 (Linux x86_64) FxSync/1.70.0.20190720144906.desktop
1566773021763	Sync.Service	INFO	Starting sync at 2019-08-26 00:43:41 in browser session deUcwphwd4Qw
1566773021764	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1566773021778	Sync.Status	INFO	Resetting Status.
1566773021778	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1566773021783	Sync.BrowserIDManager	INFO	Getting keys
1566773021785	Sync.BrowserIDManager	INFO	Getting an assertion from: https://token.services.mozilla.com/1.0/sync/1.5
1566773021785	FirefoxAccounts	DEBUG	enter getAssertion()
1566773021786	FirefoxAccounts	DEBUG	getKeypairAndCertificate: already have keyPair and certificate
1566773021786	FirefoxAccounts	DEBUG	getAssertionFromCert
1566773021788	FirefoxAccounts	DEBUG	getAssertionFromCert returning signed: true
1566773021790	Sync.BrowserIDManager	DEBUG	Getting a token
1566773021790	Services.Common.TokenServerClient	DEBUG	Beginning BID assertion exchange: https://token.services.mozilla.com/1.0/sync/1.5
1566773021791	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1566773021791	Services.Common.RESTRequest	TRACE	HTTP Header accept: application/json
1566773021791	Services.Common.RESTRequest	TRACE	HTTP Header authorization: ***** (suppressed)
1566773021791	Services.Common.RESTRequest	TRACE	HTTP Header x-client-state: ***** (suppressed)
1566773021796	Services.Common.RESTRequest	TRACE	onStartRequest: GET https://token.services.mozilla.com/1.0/sync/1.5
1566773021797	Services.Common.RESTRequest	TRACE	Channel for GET https://token.services.mozilla.com/1.0/sync/1.5 returned status code 2152398878
1566773021797	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1566773021797	Services.Common.RESTRequest	TRACE	GET body: 
1566773021798	Sync.BrowserIDManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.js:35:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.js:58:16
getTokenFromBrowserIDAssertion@tokenserverclient.js:244:13
1566773021798	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1566773021798	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1566773021799	Sync.BrowserIDManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.js:35:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.js:58:16
getTokenFromBrowserIDAssertion@tokenserverclient.js:244:13
1566773021801	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1566773021801	FirefoxAccounts	DEBUG	getOAuthToken enter
1566773021803	FirefoxAccounts	TRACE	getCachedToken returning cached token
1566773021803	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1566773021803	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1566773021803	Services.Common.RESTRequest	TRACE	HTTP Header authorization: ***** (suppressed)
1566773021803	Services.Common.RESTRequest	TRACE	HTTP Header accept: application/json
1566773021803	Services.Common.RESTRequest	TRACE	HTTP Header if-none-match: "77583deafaa9fd7b17befa167e0ec386792d8cf1-gzip"
1566773021804	Sync.Resource	WARN	GET request to https://sync-592-us-west-2.sync.services.mozilla.com/1.5/120449125/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1566773021804	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1566773021804	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1566773021805	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1566773021805	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1566773021813	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
